migration.md

```diff
@@ -1,118 +1,128 @@
-
-You can integrate this command in your build/deployment process so instrumentations are installed every time your project is deployed
-or you can run this once during development and add all the installed packages as dependencies. The command also supports printing out
-the list of instrumentation packages instead of automatically installing them. For example,
-
-```
-splk-py-trace-bootstrap -a=requirements
-
-opentelemetry-instrumentation-falcon>=0.13b0
-opentelemetry-instrumentation-requests>=0.8b0
-opentelemetry-instrumentation-sqlite3>=0.11b0
-opentelemetry-exporter-zipkin>=0.14b0
-```
-
-## Migrate configuration
-
-### General Config
-
-You must rename the following config environment variables
-
-| Old Environment Variable           | New Environment Variable             |
-| ---------------------------------- | ------------------------------------ |
-| SIGNALFX_SERVICE_NAME              | SPLK_SERVICE_NAME                    |
-| SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_ZIPKIN_ENDPOINT        |
-| SIGNALFX_RECORDED_VALUE_MAX_LENGTH | SPLK_MAX_ATTR_LENGTH                 |
-
-
-### Django
-
-#### Remove `signalfx_tracing` from INSTALLED_APPS
-
-splunk-opentelemtry no longer ships a Django app and does not require you to add anything to your `INSTALLED_APPS` or any other
-setting from settings.py
-
-#### Set `DJANGO_SETTINGS_MODULE` environment variable
-
-splunk-opentelemetry requires `DJANGO_SETTINGS_MODULE` environment to be set before it can auto-instrument a Django project. This
-should be the same as the value set in your manage.py or wsgi.py app and should be a valid Python import path.
+Because the SignalFx Tracing Library for Python uses OpenTracing and the Splunk Distribution
+of OpenTelemetry Python uses OpenTelemetry, the semantic
+conventions for span tag names change when you migrate. For more information,
+see [Migrate from OpenTracing to OpenTelemetry](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html#apm-opentelemetry-migration).
 
 ## Known issues
 
-There are some known issues when migrating from signalfx-python-tracing to splunk-opentelemetry. These will be fixed in upcoming releases.
-
-- OpenTelemetry's psycopg2 has a bug that prevents the driver from functioning correctly. Thus it has been disabled temporarily in this package.
-- Does not yet support sending spans directly to the SignalFx backend and requires the SignalFx Smart Agent or OpenTelemetry collector.
-- Does not yet support enabling or disabled specific instrumentations.
-- Does not yet support tracing and log correlation. 
\ No newline at end of file
+These are the known issues as a result of migrating from the SignalFx Tracing Library for Python to the Splunk Distribution of OpenTelemetry Python:
+
+- psycopg2 is disabled because it contains a bug that prevents the driver from functioning correctly.
+- You can't enable or disable specific instrumentations.
+- You can't inject trace context in logs.
```

Comment:
We should clarify that these features be added soon.

Reply:
i updated the known issues about features to say you can't do something "right now," but don't want to explicitly say they will be added soon in the migration doc.